wf/en-us/content/topics/workflows/connector-reference/azuread/actions/readgroup.htm

```diff
@@ -487,13 +492,13 @@ <h2>Output</h2>
                                                                     <td class="TableStyle-standard-horizonal-borders-only-BodyE-Column1-Body1">
                                                                         <ul>
                                                                             <li>
-                                                                                <p><span class="code">true</span> if the signed-in user is subscribed to receive email conversations</p>
+                                                                                <p><span class="state">true</span> if the signed-in user is subscribed to receive email conversations</p>
                                                                             </li>
                                                                             <li>
-                                                                                <p> <span class="code">false</span> otherwise.</p>
+                                                                                <p><span class="state">false</span> otherwise.</p>

```

Comment:
```suggestion
                                                                                <p><span class="state">false</span> otherwise</p>

```

Reply:
Changes made in the source.